src/main/resources/config/application.yaml

```diff
@@ -6,7 +6,7 @@ spring:
 
   cloud:
     function:
-      definition: consumeSaResult;consumeSaStopped;consumeSaFailed;consumeDsResult;consumeDsStopped;consumeDsFailed;consumeBuildResult;consumeBuildStopped;consumeBuildFailed;consumeCaseImportSucceeded;consumeCaseImportFailed;consumeSensitivityAnalysisResult;consumeSensitivityAnalysisStopped;consumeSensitivityAnalysisFailed;consumeShortCircuitAnalysisResult;consumeShortCircuitAnalysisStopped;consumeShortCircuitAnalysisFailed;consumeVoltageInitResult;consumeVoltageInitStopped;consumeVoltageInitFailed;consumeLoadFlowResult;consumeLoadFlowStopped;consumeLoadFlowFailed
+      definition: consumeSaResult;consumeSaStopped;consumeSaFailed;consumeDsResult;consumeDsStopped;consumeDsFailed;consumeBuildResult;consumeBuildStopped;consumeBuildFailed;consumeCaseImportSucceeded;consumeCaseImportFailed;consumeSensitivityAnalysisResult;consumeSensitivityAnalysisStopped;consumeSensitivityAnalysisFailed;consumeShortCircuitAnalysisResult;consumeShortCircuitAnalysisStopped;consumeShortCircuitAnalysisFailed;consumeVoltageInitResult;consumeVoltageInitStopped;consumeVoltageInitFailed;consumeLoadFlowResult;consumeLoadFlowStopped;consumeLoadFlowFailed;consumeSensitivityAnalysisNonEvacuatedEnergyResult;consumeSensitivityAnalysisNonEvacuatedEnergyStopped;consumeSensitivityAnalysisNonEvacuatedEnergyFailed
```

Comment:
this list is getting worse and worse as we are adding definitions, can we change it to something like
cloud:
function:
definition:
- consumeSaResult consumeSaStopped consumeSaFailed
- consumeDsResult consumeDsStopped consumeDsFailed
- consumeBuildResult consumeBuildStopped; consumeBuildFailed
- consumeCaseImportSucceeded consumeCaseImportFailed
- consumeSensitivityAnalysisResult consumeSensitivityAnalysisStopped consumeSensitivityAnalysisFailed
- consumeShortCircuitAnalysisResult consumeShortCircuitAnalysisStopped consumeShortCircuitAnalysisFailed
- consumeVoltageInitResult consumeVoltageInitStopped consumeVoltageInitFailed
- consumeLoadFlowResult consumeLoadFlowStopped consumeLoadFlowFailed
- consumeSensitivityAnalysisNonEvacuatedEnergyResult consumeSensitivityAnalysisNonEvacuatedEnergyStopped consumeSensitivityAnalysisNonEvacuatedEnergyFailed
?

Reply:
Changed.
The only thing that works is to use double quotes with escaped newline